src/coreanimation.cs

```diff
@@ -1020,7 +1020,7 @@ interface CAAnimation : CAAction, CAMediaTiming, NSSecureCoding, NSMutableCopyin
 		#region SceneKitAdditions
 
 #if XAMCORE_2_0
-		[TV (11,0), Mac (10,13, onlyOn64: true), iOS (11,0), NoWatch]
+		[TV (11,0), Mac (10,13, onlyOn64: true), iOS (11,0, onlyOn64: true), NoWatch]
```

Comment:
This is strange and unexpected, but master has the 64 tag here:

https://github.com/xamarin/xamarin-macios/blob/master/src/coreanimation.cs#L1070

Should I revert this chunk @spouliot @rolfbjarne ?

Reply:
That's incorrect - my guess (blame can confirm) is that it's a mistake when merging `xcode9.1` back in master (since a lot of SceneKit related API updates for `onlyOn64: true` were done only in master).

Reply:
yes